stats.py

```diff
@@ -46,19 +58,19 @@ def fetch_stats(account, character_name) -> tuple[Stats, dict]:
 def iter_passives(tree, masteries, skills):
 	for h in skills['hashes']:
 		node = tree['nodes'][str(h)]
-		yield node['stats']
+		yield node['name'], node['stats']
 
 	cluster_jewel_nodes = {}
 	for jewel in skills['jewel_data'].values():
 		if 'subgraph' in jewel:
 			cluster_jewel_nodes.update(jewel['subgraph']['nodes'])
 	for h in skills['hashes_ex']:
 		node = cluster_jewel_nodes[str(h)]
-		yield node['stats']
+		yield node.get("name", ""), node['stats']
```

Comment:
do you mind changing most instances of `"` to `'` for consistency?

maybe in a follow-up PR if that's too much work. or maybe I should just set up yapf...

Reply:
can do. shouldnt be too big of a deal.